vent/io/__init__.py

```diff
@@ -1 +1,121 @@
-
+""" Module for interacting with physical and/or simulated devices installed on the ventilator.
+
+"""
+import pigpio
+import configparser
+from importlib import import_module
+from .devices.sensors import Sensor
+
+
+class Hal:

```

Comment:
I'd rather see this in its own hal.py or toplevel.py file. I think __init__.py files in python are for usually just indicating what is exported by the modules and not for code.

Reply:
Done. See commit [8caa840](https://github.com/CohenLabPrinceton/Ventilator-Dev/pull/45/commits/8caa840ad687aa52d6249233b0041e7844ec33db) in PR #45